### ISSUE TITLE
Example page

### ISSUE THREAD

**Issue by david-ma** [slavanossar/thirst-quencher#4]
The example page is a bit broken right because it puts a comment inside a comment.

You could make a better example later, but of course this is fairly low priority stuff and your current example is not that hard to follow & understand.

I've never used partials before but it's quite clear what is going on and how to use them.

**Comment by slavanossar** [comment]
Are you referring to the partial example in `index.html`? I can't seem to see where I've put a comment within a comment.

Also, I've been meaning to add a section on partials in the documentation, there's just nothing there at the moment because I never wrote anything for my work docs.
